xds/src/test/java/io/grpc/xds/internal/sds/SecretVolumeSslContextProviderTest.java

```diff
@@ -43,11 +43,11 @@
 @RunWith(JUnit4.class)
 public class SecretVolumeSslContextProviderTest {
 
-  private static final String SERVER_1_PEM_FILE = "server1.pem";
-  private static final String SERVER_1_KEY_FILE = "server1.key";
-  private static final String CLIENT_PEM_FILE = "client.pem";
-  private static final String CLIENT_KEY_FILE = "client.key";
-  private static final String CA_PEM_FILE = "ca.pem";
+  public static final String SERVER_1_PEM_FILE = "server1.pem";
```

Comment:
this is getting disorganized, can you make a utility class including other public methods used in multiple tests?

Reply:
I agree there are too many places defining the same thing. If I do that in this PR it will pull in too many files in this PR. See my comments elsewhere.

Reply:
Assuming you are okay with adding other changes to this PR I will move this stuff to the utility class.